homeassistant/components/climate/melissa.py

```diff
@@ -201,7 +183,7 @@ def update(self):
             )['controller']['_relation']['command_log']
         except KeyError:
             _LOGGER.warning(
-                'Unable to update component %s', self.entity_id)
+                'Unable to update entity %s', self.entity_id)
```

Comment:
Accessing deeply nested dicts can become fragile if something gets changed upstream, is there a way to debug such situations if needed?

Reply:
Well at this point it will at least give a warning if the key is not present, the only thing I changed is the string (this is not a component, it is a entitiy ;) )